src/TransCreator/index.jsx

```diff
@@ -1,21 +1,20 @@
 import React, { useEffect, useState } from 'react';
-import { Item, Root } from 'native-base';
+import { Root } from 'native-base';
 import { View, Alert } from 'react-native';
 import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
 import moment from 'moment';
 import { NavigationScreenPropType } from 'react-navigation';
-
-import styles from './Style';
-import PageFooter from './PageFooter';
-import DateSelector from './DateSelector';
 import { addNewTransaction } from '../Stats/actionCreator';
 import LabelGroup from './LabelGroup';
 import PageBanner from './PageBanner';
 import theme from '../Common/themeStyle';
+import Keyboard from './Keyboard';
+import { getResult } from './calculator';
```

Comment:
you should use the calculator as a hook, so lets create a useCalculator hook, which returns the calculator it self, you 
```js
const calculator = useCalculator(0); // 0 is the inital value
calculator.append(char);
calculator.getResult();
calculator.toString();
```